### DISCUSSION TITLE
got crashed on prtsc button

### DISCUSSION THREAD

**Discussion by uciharis** [regolith-linux/regolith-desktop#901] [Q&A]
hi, im using kinetic kudu version ( i was installing regolith via iso file).

everytime i used "prtsc" key, i always get crash and force me to login screen.

i got this problem too when i was using regolith with base ubuntu 22.04 LTS before

so, i have to take a screenshot from making a new keybinding from keyboard setting ( my keybinding = win+prtsc to gnome-screenshot )

thank you for reading this

**Comment by kgilmer** [comment]
Hi, if the printscreen crash causes to to return to login, likely the session crashed. This may be due to graphics driver issues.  Do you notice any crash log info in `/var/log/syslog` after hitting `prtsc`?
